core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala

```diff
@@ -179,23 +165,12 @@ class InvokerPool(
 
 object InvokerPool {
     def props(
-        f: (ActorRefFactory, String) => ActorRef,
```

Comment:
We have to pass the InvokerFactory into the supervisor from outside. Main reason is testability.